### PR TITLE
NXP-24681 Handle Directory field default value in creation form

### PR DESCRIPTION
**Pull request by beloeuvre**
Handle Directory field default value in creation form

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

File: elements/nuxeo-document-creation/nuxeo-document-creation-behavior.html [hunk 1]
```diff
@@ -91,7 +91,7 @@
         }
         schemaFetcher.path = 'path/' + this.targetPath + '/@emptyWithDefault';
         schemaFetcher.params = { type: this.selectedDocType.type };
-        schemaFetcher.headers = { properties: '*' };
+        schemaFetcher.headers = { properties: '*', "X-NXfetch.document": "properties", "X-NXtranslate.directoryEntry": "label" };
```
Comment (Gabez0r) [inline elements/nuxeo-document-creation/nuxeo-document-creation-behavior.html]: Use single instead of double quotes, please.

**Comment by Gabez0r** [comment]
👍

**Comment by afjusto** [comment]
👍

**Comment by beloeuvre** [comment]
Merci :)

**Pull request merged**